src/js/store/index.js

```diff
@@ -0,0 +1,52 @@
+import { OPERATORS } from "../utils/helpers.js";
+
+export default class Store {
+  constructor() {
+    this.observers = [];
+    this.status = "IDLE";
+    this.total = "0";
+  }
+  addObserver(...observers) {
+    observers.forEach((observer) => this.observers.push(observer));
+  }
+  observing() {
+    this.observers.forEach((observer) => observer.render());
+  }
+  updateTotal(num) {
+    if (Number(`${this.total}${num}`) > 1000) {
+      alert("세자리 숫자까지만 가능합니다.");
+      return;
+    }
+    if (this.total === "0") {
+      this.total = `${num}`;
+    } else {
+      this.total = `${this.total}${num}`;
+    }
```

Comment:
```suggestion
   this.total = this.total === "0" ? `${num}` : `${this.total}${num}`
```
이렇게 하면 조금 더 간결해 보일 수 있어요!